### ISSUE TITLE
Instruction for amateurs

### ISSUE THREAD

**Issue by radosky7** [SuslikV/cfg-cam#3]
Hey! I don’t understand how to use it, I downloaded this and what next? I have C920 and use OBS, but what should I do with these files?

**Comment by SuslikV** [comment]
Yeah, it wasn't designed for amateurs for sure. Maybe will write wiki entries later. This is cmd tool - it has no user graphical interface.

**Comment by SuslikV** [comment]
Brief info for beginners: https://github.com/SuslikV/cfg-cam/wiki/Usage

**Comment by galexmay** [comment]
The brief talks about an exe file. I downloaded and unzipped and there was no exe file

**Comment by SuslikV** [comment]
@galexmay the "WebCameraConfig_1_0.7z" file?

You can download the "WebCameraConfig.exe" too. But maybe it was eaten by your security software during downloading. I can only guess on this. I posted CRC of the .exe file (https://github.com/SuslikV/cfg-cam/releases) to be sure that the file you got is the same to one I posted.

I added releases link - https://github.com/SuslikV/cfg-cam/releases , to the wiki page mentioned below.

Brief info for a beginners: https://github.com/SuslikV/cfg-cam/wiki/Usage

**Comment by galexmay** [comment]
Thanks for your comment. I believe the addition of the link to the releases
folder will help a lot of beginners like me.
I also came across another issue which many might face. When I try and call
OBS and your programme with the same cmd or bat file, your programme
doesn't work. Do you have any idea why that might be or could you provide a
script on the beginners page to show how that can be done?

Thanks
Alex

**Comment by SuslikV** [comment]
@galexmay OBS requires some time to init the sources (and your camera device).
I think, the simplest way is to start the WebCameraConfig.exe file with a significant delay (to ensure that all sources had initialized).

The `run_all.cmd` file may look like this (7 seconds delay):
```
cd "C:\My Programs\OBS\bin\64bit\"
start "Starting OBS Sudio..." "C:\My Programs\OBS\bin\64bit\obs64.exe"
timeout 7
cd "C:\My programs\WebCameraConfig\"
"C:\My programs\WebCameraConfig\WebCameraConfig.exe" --profile night
```
I hope this script works well...

**Comment by galexmay** [comment]
Sorry for the delay in replying. I wanted to test it out fully first.
It seems to work very well. I tried a similar approach myself but failed so
I am not sure why this works better, but it does.
Can I suggest you add this to the page for beginners as well.

Thanks
Alex
